src/templates/views/partials/global-footer.hbs

```diff
@@ -40,6 +40,7 @@
 		<nav class="sponsor-nav">
 			<img class="sponsor sponsor-jordans-furniture" src="/dist/img/jordans-furniture-logo-new.svg" alt="jordans furniture">
 			<img class="sponsor sponsor-jack-williams" src="/dist/img/jack-williams-logo-new.svg" alt="jack williams">
+			<img class="sponsor sponsor-cummings-foundation" src="https://mare-public.s3.amazonaws.com/images/cummings-foundation-logo.svg" alt="cummings-foundation">
```

Comment:
Small nit, but I don't think `cummings foundation` should be hyphenated in the alt tag.

Reply:
Updated!